agent/runner/actions/query_transform.go

```diff
@@ -47,36 +48,41 @@ are needed and the pmm user is a not privileged user.
 This function converts DML queries to the equivalent SELECT to make
 it able to explain DML queries on older MySQL versions and for unprivileged users.
 */
-func dmlToSelect(query string) string {
+func dmlToSelect(query string) (string, bool) {
```

Comment:
Can you please add comment with explanation of what this bool stands for?

Reply:
Sure. I added comment to func:
// dmlToSelect returns query converted to select and boolean, if conversion were needed.

Take a look if its clear, thank you.